src/oogeso/core/devices/storage.py

```diff
@@ -247,7 +247,7 @@ def compute_costForDepletedStorage_alt2(self, pyomo_model: pyo.Model, timesteps:
         dev_data = self.dev_data
         # E_target = dev_data.E_max
         E_target = pyomo_model.paramDeviceEnergyTarget[self.id]
-        t_end = timesteps.last()
+        t_end = timesteps[-1]
```

Comment:
Could this be anything other than a List[int]?

Reply:
It's a pyo.Set (pyomo_model.setHorizon), so I think it's the type hint that is wrong.  But last() is probably just doing the same as [-1]